LoopbackApp/build.gradle

```diff
@@ -1,15 +1,17 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     repositories {
+        google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle-experimental:0.11.1'
+        classpath 'com.android.tools.build:gradle:3.1.3'
     }
 }
 
 allprojects {
     repositories {
+        google()
```

Comment:
If we're switching to cmake, why do we need google repository?  I thought that was only for gradle experimental.

Reply:
google() is for appcompat I believe.